### ISSUE TITLE
Keep Markdown reference-style links in output

### ISSUE THREAD

**Issue by adjohnson916** [verbose/verb#70]
Could we keep reference-style links in Markdown input in the Markdown output?

**Comment by jonschlinkert** [comment]
yes, you would need to disable the markdown formatting plugin. when I get a chance I'll look at verb to make sure this is (easily) possible

fwiw, if you want to use a different markdown formatter, I published two different libs for this, one based on remarkable - this is the one used by verb and it resolves reflinks, and a more naive one that uses regex.

**Comment by adjohnson916** [comment]
@jonschlinkert Thanks.

I've found one way to do this is adding the following to my `verbfile.js`:
```js
var verb = require('verb');
verb.disable('dest:format plugin');
```

I haven't found how to substitute for different format plugins yet.

I ask because it seems by default nested lists are formatting with `~` for some items, which appears to not be supported by GitHub Flavored Markdown, nor apparently is it a part of CommonMark (http://spec.commonmark.org/0.19/#list-items). I'd submit a PR to fix that if I could find the source.

**Comment by jonschlinkert** [comment]
> default nested lists are formatting with ~

That [code is here](https://github.com/jonschlinkert/list-item/blob/master/index.js#L90), pr would be great

**Comment by adjohnson916** [comment]
I've reopened this for two reasons:

1. jonschlinkert/list-item#1 was only part of the issue - I haven't decided yet if I still want to try to address the reference link output.
1. Since jonschlinkert/list-item#1 bumped to 1.0.x, it may not have propagated here yet - pending investigation.

**Comment by adjohnson916** [comment]
Update on (2) from my previous comment: jonschlinkert/list-item#1 with 1.0.x should now have propagated to this project via jonschlinkert/markdown-utils#2 as 0.7.1.
